kernel/src/main/java/org/kframework/EvaluationException.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+package org.kframework;
+
+import java.util.ArrayList;
+import org.kframework.kore.K;
+
+
+public class EvaluationException extends RuntimeException {
+    private int steps_;
+    private ArrayList<K> stacktrace_;
```

Comment:
this should be List, not ArrayList. In general, in Java you should never declare variables to be of type ArrayList or HashSet or HashMap, but instead List or Set or Map.